text/0000-macro-at-most-once-rep.md

```diff
@@ -0,0 +1,166 @@
+- Feature Name: macro-at-most-once-rep
+- Start Date: 2018-01-17
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+
+Summary
+-------
+
+Add a repetition specifiers to macros to repeat a pattern at most once: `$(pat)?`.  `?` behaves like `+` or `*` but represents at most one repetition of `pat`.
+
+Motivation
+----------
+
```

Comment:
Another motivation you could add is an argument from familiarity with regular expressions.

Reply:
I mention this in the rationale as a reason why `?` was chosen, but since the RFC explicitly chooses to _not_ add `{M, N}`, I think it would be hard to add familiarity with regexes to the motivation.